projects/netgrif-components-core/src/lib/task/services/assign-policy.service.ts

```diff
@@ -49,6 +51,21 @@ export class AssignPolicyService extends TaskHandlingService {
      * @param afterAction the action that should be performed when the assign policy (and all following policies) finishes
      */
     public performAssignPolicy(taskOpened: boolean, afterAction: AfterAction = new AfterAction()): void {
+        if (!this._userService.isUserEmpty(this._userService.user)) {
```

Comment:
might be also nice to have a `isCurrentUserEmpty` method

Reply:
Done